### PR TITLE
Adds reusable version command

### PR DESCRIPTION
**Pull request by krishvoor**
## Type of change

- [ ] Refactor
- [x] New feature
- [ ] Bug fix
- [ ] Optimization
- [ ] Documentation Update

## Description
This PR adds a reusable version, that can be integrated into
1) k8s-netperf
2) ingress-perf
3) kube-burner

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please describe the System Under Test.
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

File: version/version.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
+// Copyright 2020 The Kube-burner Authors.
```
Comment (rsevilla87) [inline version/version.go:1]: Update header pls :)

Comment (krishvoor) [reply version/version.go:1]: I kept it "as-is" so not to mess with "Kube-burner" Orginal Authors :P

Comment (vishnuchalla) [reply version/version.go]: @krishvoor We have our own header for `go-commons` btw: https://github.com/cloud-bulldozer/go-commons/blob/main/comparison/elastic.go#L1 😛

Comment (krishvoor) [reply version/version.go]: Updated, ptal

**Review by vishnuchalla — APPROVED** [review]

**Pull request merged**